specification/search/Azure.Search/models-service.tsp

```diff
@@ -0,0 +1,9176 @@
+     */
+    @visibility("read")
+    @encodedName("application/json", "value")
+    dataSources: SearchIndexerDataSource[];
+}
+
+/**
+ * Represents an indexer.
+ */
+model SearchIndexer {
+    #suppress "@azure-tools/typespec-azure-core/no-closed-literal-union" "return status code"
+    @doc("Succesfully created or updated")
+    @statusCode _: 200 | 201;
+    /**
+     * The name of the indexer.
+     */
+    name: string;
+
+    /**
+     * The description of the indexer.
+     */
+    description?: string;
+
+    /**
+     * The name of the datasource from which this indexer reads data.
+     */
+    dataSourceName: string;
+
+    /**
+     * The name of the skillset executing with this indexer.
+     */
+    skillsetName?: string;
+
+    /**
+     * The name of the index to which this indexer writes data.
+     */
+    targetIndexName: string;
+
+    /**
+     * The schedule for this indexer.
+     */
+    schedule?: IndexingSchedule;
+
+    /**
+     * Parameters for indexer execution.
+     */
+    parameters?: IndexingParameters;
+
+    /**
+     * Defines mappings between fields in the data source and corresponding target
+     * fields in the index.
+     */
+    fieldMappings?: FieldMapping[];
+
+    /**
+     * Output field mappings are applied after enrichment and immediately before
+     * indexing.
+     */
+    outputFieldMappings?: FieldMapping[];
+
+    /**
+     * A value indicating whether the indexer is disabled. Default is false.
+     */
+    @encodedName("application/json", "disabled")
+    isDisabled?: boolean;
+
+    /**
+     * The ETag of the indexer.
+     */
+    @encodedName("application/json", "@odata.etag")
+    eTag?: string;
+
+    /**
+     * A description of an encryption key that you create in Azure Key Vault. This key
+     * is used to provide an additional level of encryption-at-rest for your indexer
+     * definition (as well as indexer execution status) when you want full assurance
+     * that no one, not even Microsoft, can decrypt them. Once you have encrypted your
+     * indexer definition, it will always remain encrypted. The search service will
+     * ignore attempts to set this property to null. You can change this property as
+     * needed if you want to rotate your encryption key; Your indexer definition (and
+     * indexer execution status) will be unaffected. Encryption with customer-managed
+     * keys is not available for free search services, and is only available for paid
+     * services created on or after January 1, 2019.
+     */
+    encryptionKey?: SearchResourceEncryptionKey;
+}
+
+/**
+ * Represents a schedule for indexer execution.
+ */
+model IndexingSchedule {
+    /**
+     * The interval of time between indexer executions.
+     */
+    interval: duration;
+
+    /**
+     * The time when an indexer should start running.
+     */
+    // FIXME: (utcDateTime) Please double check that this is the correct type for your scenario.
```

Comment:
Is this comment still necessary?

Reply:
https://learn.microsoft.com/azure/search/search-howto-schedule-indexers?tabs=portal#schedule-definition

`utcDateTime` appears to be the correct choice for this property